src/.nuget/toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development</id>
```

Comment:
Essentially without a RID in the graph NuGet will only pick up the explicit RID.  So you'd get assets for ubuntu.14.04-arm but no assets for "unix" or "linux" etc since NuGet wouldn't know that ubuntu.14.04-arm also includes these RIDs.  You should first make the changes to the runtime.json in the platforms package then make these.  There are quicker ways to do it (EG: shipping a runtime.json in this package) but I'd like to avoid those.  @schellap let me know if you have questions.